src/style.css

```diff
@@ -1,3 +1,74 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
```

Comment:
From the checklist: 'Don't use * selector (it impacts performance)'. Set styles only for elements that require them. Zeroing out your margins, paddings or other styles with '*' is still inefficient for browser.